finetuner-workflow/finetuner/requirements.txt

```diff
@@ -1,13 +1,7 @@
-pandas~=1.3.5
-transformers==4.25.1
-deepspeed~=0.7.6
-tokenizers~=0.13.2
-huggingface-hub~=0.11.1
-numpy~=1.22.3
-pynvml~=11.4.0
-wandb~=0.12.10
-torch~=1.13.0
-psutil~=5.9.0
-validators
-requests
-accelerate~=0.14.0
+transformers~=4.27.2
+deepspeed==0.8.3
```

Comment:
This should be updated to 0.9.0 for PyTorch 2.0 support, since certain version asserts in 0.8.3 break with PyTorch 2.0.

Reply:
Done. Let's see if this works.